### PR TITLE
docs: add io.cozy.accounts 📚

### PR DESCRIPTION
**Pull request by gregorylegarec**
This PR is in WIP as we need to extract `accountName` from the `auth` attributes and rename it `label`.

### CODE REVIEW

File: docs/io.cozy.accounts.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+## `io.cozy.accounts`
+
+The `io.cozy.accounts` doctype stores authentification informations used by konnectors to connect to external services or API.
+
+Its attributes are:
+
+- `account_type` (_deprecated_) : {string} Slug of the konnector the account is related to.
+- `auth`: {object} Contains authentification data, typically a couple with `login`/`password`. It could also contain an attribute `token` for OAuth konnectors.
+- `label`: {string} Label given by user.
+- `data`: {object} Additional data.
```
Comment (CPatchane) [inline docs/io.cozy.accounts.md]: Do we have some use case for example?

File: docs/io.cozy.accounts.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# Cozy Accounts doctype
+
+## `io.cozy.accounts`
+
+The `io.cozy.accounts` doctype stores authentification informations used by konnectors to connect to external services or API.
+
+Its attributes are:
+
+- `account_type` (_deprecated_) : {string} Slug of the konnector the account is related to.
+- `auth`: {object} Contains authentification data, typically a couple with `login`/`password`. It could also contain an attribute `token` for OAuth konnectors.
```
Comment (CPatchane) [inline docs/io.cozy.accounts.md]: It contains all values provided by the konnectors fields also.

**Comment by kosssi** [comment]
@gregorylegarec on fait quoi ?

**Comment by ptbrowne** [comment]
I think we can merge, @CPatchane ?

**Comment by ptbrowne** [comment]
ping @gregorylegarec @CPatchane ?

**Comment by gregorylegarec** [comment]
[(2) Documenter les docTypes](https://trello.com/c/LdOIlUq0/899-2-documenter-les-doctypes)

**Comment by gregorylegarec** [comment]
Should be ok now (ping @CPatchane, @kosssi, @ptbrowne)

**Comment by CPatchane** [comment]
Thanks @gregorylegarec 🎢

**Pull request merged**